### PR TITLE
VOX UPDATE

### PR DESCRIPTION
**Pull request by Iamgoofball**
Adds a tool from /vg/ for generating VOX sound effects, and adds some more words to the listings.

Changes the VOX voice to a male voice. This voice sounds way better then the female voice we were using before, and is easier to hear and understand.

In order to add a new word, all I have to do is add it to voxwords.txt, compile it locally again, and then commit the new word.

### CODE REVIEW

**Comment by ghost** [comment]
Can you change the AI announcements to the male voice? The attention: sound, the shuttle call, the welcome to the station sound?

**Comment by Iamgoofball** [comment]
Will look into it.

File: tools/vox/voxwords.txt [hunk 1]
```diff
@@ -0,0 +1,842 @@
...
+ling
+nuke
+escape
+crew
+cultist
+cultists
+dorms
+erotic
+roleplay
+emag
```
Comment (Atskadan) [inline tools/vox/voxwords.txt:835]: i dont think the names of traitor items is a good idea
it encourages ais to antag hunt and meta, which they shouldnt be doing as a neutral party

Comment (Iamgoofball) [inline tools/vox/voxwords.txt:835]: Alright, I can remove that.

**Comment by GunHog** [comment]
Feh, just give us a speech engine already.

**Comment by Iamgoofball** [comment]
You want a side of lag w/ that sir?

**Comment by ghost** [comment]
I would love the game to lag more so the AI can rarely use an even more obnoxious announcement system

**Comment by Atskadan** [comment]
so
or
else
guy/guys
happy
sad
angry
mad
otherwise
PEOPLE
name
very
person
put
as
board
who
signed
director
leader
dear
delivered
gear
which
can
bar
kitchen
gateway
since
impede/s
productive
play/ing
sold
brig
alarmed
negative
affirmative
longer
camera/s
prison
wing

all job names

thief
attack
bomb
hot 
cold
atmospherics
wire
robot
air
tank
nitrous
oxide
oxygen
carbon
dioxide
window
botany
botanist
courtroom
court
dorms
wardrobe
bed
airlock
chair
vendor
SPACE
paper
plating
table
fix
light/s
around
telecommunications
radio

**Comment by Atskadan** [comment]
@Iamgoofball add those

**Comment by Atskadan** [comment]
> @Iamgoofball add those

**Comment by Iamgoofball** [comment]
calm down, i've been doing shit recently

**Comment by Atskadan** [comment]
to me it looks like youve been doing a whole lot of nothing

**Comment by Iamgoofball** [comment]
on my list of priorities as a human being, 2d spaceman is one of the lower ones.

i'll get back to this when I have time.

**Pull request closed without merging**